pkg/controller/rmdworkload/rmdworkload_controller.go

```diff
@@ -178,8 +179,8 @@ func (r *ReconcileRmdWorkload) findObseleteWorkloads(request reconcile.Request)
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	obseleteWorkloads := make(map[string]string)
 
-	for nodeName, namespace := range r.rmdNodeData.RmdNodeList {
-		address, err := r.getPodAddress(nodeName, namespace)
+	for _, nodeName := range r.rmdNodeData.RmdNodeList {
+		address, err := r.getPodAddress(nodeName, defaultNamespace)
```

Comment:
`getPodAddress` function should only accept one parameter (nodeName) as we are now assuming it will always be default. The `defaultNamespace` const can be used inside the function instead.